Base/src/main/java/io/deephaven/base/CompareUtils.java

```diff
@@ -159,7 +159,7 @@ public static boolean doubleNotEquals2(double x, double y, double tolerance) {
     }
 
     /**
-     * You can't do double1 > double2 because floating point numbers are not exact values. Just make sure that x-y is
+     * You can't do double1 &gt; double2 because floating point numbers are not exact values. Just make sure that x-y is
```

Comment:
It seems like this should be:
```suggestion
     * You can't do {@code double1 > double2} because floating point numbers are not exact values. Just make sure that x-y is
```
Same comment many places that `&gt` or `&lt` are used.  It seems like these symbols just indicate that we are in "code".  In my quick look, every time I see these changes, we are in code.

Reply:
Unless I'm very much mistaken (will validate by changing the code to confirm), html is allowed inside of `@code` tags, so this will not work. It might separately be a good idea to wrap in `@code`, but I'm not reading content beyond "is it less wrong than it was before".

Reply:
I agree on not holding up for better.  Javadocs need real work at some point.

Reply:
I am mistaken! The code tags do let this work, but I'm going to leave it as-is for now, we can revert/fix later once we're locked in with "this will build correctly" and have time for more specific review.